### PR TITLE
Separated general functions

### PR DESCRIPTION
**Pull request by Joshhw**
Hey Yall, I managed to move the general functionality to its own folder structure. Can someone please pull this in and test it.

### CODE REVIEW

**Comment by jmartini** [comment]
Checking this out.... (slack seems to be down)

**Comment by jmartini** [comment]
I used deploy_tools to zip this up and put it on an Amazon Lambda instance. When using the Alexa service simulator, the lambda server produces the following error:

```Unable to import module 'lambda_function': attempted relative import with no known parent package```

File: BostonData/Intents/__init__.py [hunk 1]
```diff
@@ -0,0 +1 @@
+__all__ = ["GetAddressIntent", "SetAddressIntent", "TrashDayIntent", "WorkZonesOnAnyStreetIntent", "WorkZonesOnMyStreetIntent"]
```
Comment (jmartini) [inline BostonData/Intents/__init__.py]: It doesn't look like these are being used at all

Comment (Joshhw) [reply BostonData/Intents/__init__.py]: That was left over from me messing with sarat's pull. I'll fix this up and continue testing it.

**Pull request closed without merging**